app/services/hesa/backfilling.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Hesa
+  class Backfilling
```

Comment:
Love that we have this in a service now!

Can we name/namespace this something to do with HESA degrees? Reads to me like it'd be calling `Trainees::CreateFromHesa` rather than `Degrees::CreateFromHesa`. And we might need a similar thing for trainee attributes soon!

Maybe `Hesa::BackfillDegrees` ?

Reply:
good idea 👍

Reply:
done